src/rabbit_misc.erl

```diff
@@ -1096,6 +1098,17 @@ group_proplists_by(KeyFun, ListOfPropLists) ->
                       end, #{}, ListOfPropLists),
     [ X || {_, X} <- maps:to_list(Res)].
 
+% get the full path to the erl executable used to start this VM
+get_erl_path() ->
+    {ok, [[Root]]} = init:get_argument(root),
+    Bin = filename:join(Root, 'bin'),
+    case os:type() of
+        {win32, _} ->
+            {ok, filename:join(Bin, 'erl.exe')};
```

Comment:
Doesn't `erl` without the file extension works on Windows as well? It would simplify the code even further.

Reply:
We probably don't need an atom to store the executable name or `bin` :-)

Reply:
OK I'll use a string.

I don't believe that `erl` on its own will work on Windows, but I can test that.

Reply:
Actually we must use `erl.exe` to avoid any search for an executable using the `PATHEXT` extensions. The shell started by `os:cmd` to run the epmd-starter would probably use `erl.bat` before `erl.exe` if we didn't use an extension here.